### PR TITLE
Added font download stage in github actions

### LINKED ISSUE #5406: Create GitHub actions to download fonts from AWS
### Description

Create a GitHub actions to download fonts from AWS for building electron application

### Acceptance criteria

- Upload fonts to private s3 bucket
- Download all fonts from s3 bucket using AWS credential.
- Should be able to build application without errors

### PR DESCRIPTION
**Pull request by abhishekjalan87**
### What was the problem?

Unable to download fonts on local directory - Closes #5406

### How was it solved?

Uploaded in private s3 bucket. Downloading all fonts from s3 bucket using AWS credential.

### How was it tested?

Tested on our local system.

### CODE REVIEW

**Review by eniolam1000752 — APPROVED** [review]

File: .github/workflows/electron-build.yml [hunk 1]
```diff
@@ -23,6 +23,26 @@ jobs:
         with:
           node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
 
+      - uses: actions/setup-python@v4 #Install awscli on all platform
+        with:
+          python-version: '3.10'
+      - name: Install dependencies
```
Comment (ManuGowda) [inline .github/workflows/electron-build.yml]:
```suggestion
      - name: Install AWS dependencies
```

File: .github/workflows/electron-build.yml [hunk 2]
```diff
@@ -23,6 +23,26 @@ jobs:
...
+          pip install awscli
+
+      - name: Configure AWS cli
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+
+      - name: Download from S3
```
Comment (ManuGowda) [inline .github/workflows/electron-build.yml]:
```suggestion
      - name: Download fonts from S3
```

File: .github/workflows/electron-build.yml (hunk 1 above)
Comment (abhishekjalan87) [reply .github/workflows/electron-build.yml]: Name changed

File: .github/workflows/electron-build.yml (hunk 2 above)
Comment (abhishekjalan87) [reply .github/workflows/electron-build.yml]: Name changed

**Review by ManuGowda — APPROVED** [review]

**Review by Nazgolze — APPROVED** [review]

**Pull request merged**